tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = pypy, pypy3, py27, py34, py35, py36
+envlist = pypy, pypy3, py27, py34, py35, py36, flake8
 
 [testenv]
 deps = -rrequirements-dev.txt
 commands =
     py.test sasstests.py
+
+[testenv:flake8]
+deps= flake8
+skip_install=True
+commands=
     flake8 .
```

Comment:
I'm also not sure this actually speeds things up.  In experience, usually *decreasing* the number of tox environments is the best way to speed things up.  `setup.cfg` does need fixing it seems to make flake8 even pass in tox with this patch:

```diff
diff --git a/setup.cfg b/setup.cfg
index e3a857c..5c4fcd4 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -3,4 +3,4 @@ upload_doc = build_sphinx upload_doc
 release = sdist upload build_sphinx upload_doc
 
 [flake8]
-exclude = build,dist,docs,ez_setup.py,upload_appveyor_builds.py
+exclude = .tox,build,dist,docs,ez_setup.py,upload_appveyor_builds.py
```

After that, `flake8 .` returns nearly instantly

I'll try and get some numbers for this to see if it's even worth it